docs/decisions/0022-spend-limits-constraint.rst

```diff
@@ -0,0 +1,75 @@
+0022 Constraint on ``spend_limit`` based on ``total_deposits``
+********************************************
+
+Status
+======
+**Accepted** (June 2024)
+
+Context
+=======
+When a customer provisions a learner-credit-based policy, ``enterprise_subsidy``'s ``Subsidy`` model populates the field 
+``starting_balance`` with their initial deposit. Depending on which tool was used to provision the customer's subsidy,
+the ``starting_balance`` would initally be set as the sum of all policies ``spend_limit`` or a user defined ``starting_balance``.
+A pattern began to emerge where the the sum of all policies associated to a subsidy's ``spend_limit`` field found on the 
+``SubsidyAccessPolicy`` model would be greater then the ``total_deposits`` value. The ``total_deposits`` value represents the sum of
```

Comment:
nit: `total_deposits` is really only the _positive_ committed transactions, right?

Reply:
...or really, just the sum of all committed deposits and adjustments.